test/storage-luatest/storage_1_test.lua

```diff
@@ -97,3 +97,29 @@ test_group.test_sharded_spaces = function(g)
         s1:drop()
     end, {g.params.engine})
 end
+
+test_group.test_simultaneous_cfg = function(g)
+    g.replica_1_a:exec(function(cfg)
+        _G.ivshard.storage.internal.errinj.ERRINJ_CFG_DELAY = true
```

Comment:
You don't need `_G` for `ivshard` and other `i...` variables - they are explicitly allowed globals in our `.luacheckrc` file. For `fiber_cfg` you need it because it is not ignored in luacheck config, but for `ivshard`, `ifiber`, `iwait_timeout`, `iverror` and others you don't.

Here and in all similar places in the patchset.

Reply:
Thank you for explanation